### DISCUSSION TITLE
/fetch error

### DISCUSSION THREAD

**Discussion by yordyi** [zed-industries/zed#18560] [Feature Requests]
<img width="608" alt="image" src="https://github.com/user-attachments/assets/996ea0a1-d7ad-46b3-abc0-0c455523f83f">
## Don't you support utf8? Chinese can't be parsed.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
